ml/pre-processing/get_VT_signatures.scala

```diff
@@ -0,0 +1,45 @@
+import com.datastax.spark.connector._
+import play.api.libs.json.Json
+import java.io.{ByteArrayOutputStream, ByteArrayInputStream}
+import java.util.zip.{GZIPOutputStream, GZIPInputStream}
+
+case class VT_results_by_service_name_class(service_name: String, sha256: String)
+case class VT_results_by_sha256_class(sha256: String, service_name: String, results: Array[Byte] )
+case class VT_join_results_class(sha256: String, service_name: String, results: String)
+case class VT_sample_signatures_initial_seq_rdd_class(sha256: String, seq_results: Seq[String])
+case class VT_sample_signatures_final_array_rdd_class(sha256:String, array_results:Array[Double])
+
+def unzip(x: Array[Byte]) : String = {		
+    val inputStream = new GZIPInputStream(new ByteArrayInputStream(x))
+    val output = scala.io.Source.fromInputStream(inputStream).mkString
+    return output
+}
+def deleteNumberInSampleSignatures(x: String): Boolean = {
+    val regex = "[0-9]".r
+    return regex.findFirstIn(x).isEmpty
+}
+
+val VT_results_by_service_name_meta = sc.cassandraTable[VT_results_by_service_name_class]("gsoc3","results_meta_by_service_name").where("service_name=?","virustotal")
```

Comment:
Thanks, I got it! I add the config file now. And I also change the code, replacing the keyspace, table, and file directories with parameters in config file.